build-python.sh

```diff
@@ -352,26 +356,26 @@ if test "${BUILD_WHEEL}" = true; then
     python -m build \
         --wheel \
         --outdir ../dist \
-        ${BUILD_ARGS} \
+        ${BUILD_ARGS:+ -o "$BUILD_ARGS"} \
         .
 fi
 
 if test "${INSTALL}" = true; then
     echo "--- installing lightgbm ---"
     cd ../dist
     if test "${BUILD_WHEEL}" = true; then
-        PACKAGE_NAME="lightgbm*.whl"
+        PACKAGE_NAME=$(find . -name "lightgbm*.whl" -print -quit)
```

Comment:
```suggestion
        PACKAGE_NAME="$(echo lightgbm*.whl)"
```

I think `echo` would be simpler here.